automatic/git.portable/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference = 'Stop'
+
+$installLocation = Join-Path $(Get-ToolsLocation) 'git'
```

Comment:
@majkinetor What is considered best practices here? Honstely I just ported this from the current implementation (just replaced the deprecated function with `Get-ToolsLocation`)